package-lock.json

```diff
@@ -4698,10 +4697,71 @@
         "gl-state": "^1.0.0",
         "gl-vao": "^1.1.1",
         "gl-vec4": "^1.0.1",
-        "glslify": "^6.1.0",
+        "glslify": "^6.4.1",
         "robust-orientation": "^1.1.3",
         "split-polygon": "^1.0.0",
         "vectorize-text": "^3.2.0"
+      },
+      "dependencies": {
+        "glslify": {
```

Comment:
Hmm. That seems off. Looks like `glslify` didn't get dedupe.

Can you run `npm ls glslify` and paste its result here?